tests/agent/__init__.py

```diff
@@ -0,0 +1,48 @@
+from contextlib import contextmanager
+import requests
+import time
+from urlparse import urljoin
```

Comment:
```suggestion
from urllib.parse import urljoin
```

Reply:
see https://stackoverflow.com/questions/48072619/how-can-i-import-urlparse-in-python-3?answertab=votes#tab-top